adafruit_trellism4.py

```diff
@@ -82,7 +84,18 @@ def __setitem__(self, index, value):
     @property
     def brightness(self):
         """
-        Overall brightness of the pixel.
+        The overall brightness of the pixel. Must be a number between 0 and
+        1, where the number represents a percentage between 0 and 100, i.e. ``0.3`` is 30%.
+
+        This example turns all the LEDs red and sets the brightness to ``0.3``:
+
+        .. code-block:: python
+
+            from adafruit_trellism4 import trellis
+
+            trellis.pixels.brightness = 0.3
```

Comment:
This example does not actually use the top-level property: it operates on the `.pixels` property.

Reply:
Correct. But so do the rest of the things here. As stated above, this documentation doesn't show up in the docs anyway.

Reply:
never mind, per above